wallet-new/src/Cardano/Wallet/Server/Plugins.hs

```diff
@@ -161,7 +162,7 @@ legacyWalletBackend pm txpConfig WalletBackendParams {..} ntpStatus = pure $ \di
 
     exceptionHandler :: SomeException -> Response
     exceptionHandler se =
-        case asum [handleV1Errors se, handleV0Errors se] of
+        case asum [handleV1Errors se, handleWalletException <$> fromException se, handleV0Errors se] of
```

Comment:
Different styles here, I like it better though

Could make all handlers `e -> Response` too.